phalcon/Validation/Validator/Url.zep

```diff
@@ -57,14 +57,20 @@ class Url extends Validator
      */
     public function validate(<Validation> validation, var field) -> bool
     {
-        var value, message, label, replacePairs, code;
+        var code, label, message, options, replacePairs, result, value;
 
         let value = validation->getValue(field);
 
-        if !filter_var(value, FILTER_VALIDATE_URL) {
-            let label = this->prepareLabel(validation, field),
+        if fetch options, this->options["flags"] {
+            let result = filter_var(value, FILTER_VALIDATE_URL, ["flags" : options]);
```

Comment:
How about 'options' as well ? there is also missed ability to validate like that:
```php
filter_var(
    'http://example.com',
    FILTER_VALIDATE_URL,
    FILTER_FLAG_PATH_REQUIRED // not array
);
```

Reply:
Fixed